docs/3____physical_signal_abstraction.adoc

```diff
@@ -8,7 +8,8 @@ Variables representing these physical signals are clocked to reflect the bus tim
 Changes to these clocked variables always reach the destination.
 
 The figure below shows an example architecture of a high-cut.
-Here the signals (_Signal 1...8_) from PDUs, which in turn are assigned to frames, are modeled as clocked FMU variables of a specific type.
+The signals (_Signal 1...8_) are modeled as clocked FMU variables of a specific type.
```

Comment:
what does "of a specific type" mean here? is this part of the sentence even necessary?

Reply:
I added this sentence to show that for example if the specified signal is of type float32 the fmi3 variable is also of type float32. Just to introduce the reader at this point the first time. Do you think this is to much information at this point of the document? But I´m also fine with removing "of a specific type".

Reply:
@klausschuch I'm going to merge that PR now so I can get it completed. It has now been open for a total of more than two months...